### ISSUE TITLE
[SF][A11y][ObjectStatus]: Unnecessary and confusing information is announced

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents-react#7700]
### Describe the bug

Screen reader is unnecessarily announcing the "Indication Color 10" information along with the text.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-objectstatus--docs

### Reproduction steps

1. Goto https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-objectstatus--docs
2. Set "state" to "Indication10" and "interactive" to true 
3. Focus to Object Status component
4. Observe JAWS's annoucement

Actual:
JAWS announce "ObjectStatus Indication Color 10 Object Status Button"

Expected:
JAWS should not announce unnecessary and confusing info like "Indication Color 10"

### Screenshots or Videos

<img width="1138" height="590" alt="Image" src="https://github.com/user-attachments/assets/81b7a952-77e4-4d3e-980d-e351d17262c1" />

### UI5 Web Components for React Version

2.13

### UI5 Web Components Version

2.13

### Browser

Chrome

### Operating System

macOS

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @2bno1 

this behavior is by design. Also, the same announcements are made for the `sap.m.ObjectStatus` ([docs](https://ui5.sap.com/#/entity/sap.m.ObjectStatus/sample/sap.m.sample.ObjectStatus)).

**Comment by judith9209** [comment]
@Lukas742 In https://ui5.sap.com/#/api/sap.m.ObjectStatus you have stateAnnouncementText that overrides those useless texts. Please reopen the defect and provide same feature as in BGSOFUIBALKAN-6498.

**Comment by Lukas742** [comment]
@judith9209 please refer to the docs.`stateAnnouncementText` is available for the React `ObjectStatus` as well.

**Comment by judith9209** [comment]
@Lukas742  True. Thanks for mentioning it.
